integration/client/container_fuzz_test.go

```diff
@@ -405,44 +408,44 @@ func doFuzz(data []byte, shouldTearDown bool) int {
 // This fuzzer is experimental for now and is being run
 // continuously by OSS-fuzz to collect feedback on
 // its sustainability.
-func FuzzIntegNoTearDownWithDownload(data []byte) int {
-	if !haveInitialized {
-		shouldRestart := initInSteps()
-		if shouldRestart {
-			return 0
+func FuzzIntegNoTearDownWithDownload(f *testing.F) {
+	f.Fuzz(func(t *testing.T, data []byte) {
+		if !haveChangedDownloadPATH {
+			if shouldRestart := initInSteps(t); shouldRestart {
+				return
+			}
 		}
-	}
-	ret := doFuzz(data, false)
-	return ret
+		doFuzz(t, data, false)
+	})
 }
 
 // FuzzIntegCreateContainerNoTearDown() implements a fuzzer
-// similar to FuzzIntegCreateContainerWithTearDown() with
+// similar to FuzzIntegCreateContainerWithTearDown()
 // with one minor distinction: One tears down the
 // daemon after each iteration whereas the other doesn't.
 // The two fuzzers' performance will be compared over time.
-func FuzzIntegCreateContainerNoTearDown(data []byte) int {
-	if !haveInitialized {
-		err := updatePathEnv()
-		if err != nil {
-			return 0
+func FuzzIntegCreateContainerNoTearDown(f *testing.F) {
+	f.Fuzz(func(t *testing.T, data []byte) {
+		if !haveChangedOSSFuzzPATH {
+			if err := updatePathEnv(); err != nil {
+				return
+			}
 		}
-	}
-	ret := doFuzz(data, false)
-	return ret
+		doFuzz(t, data, false)
+	})
 }
 
 // FuzzIntegCreateContainerWithTearDown() is similar to
 // FuzzIntegCreateContainerNoTearDown() except that
 // FuzzIntegCreateContainerWithTearDown tears down the daemon
 // after each iteration.
-func FuzzIntegCreateContainerWithTearDown(data []byte) int {
-	if !haveInitialized {
-		err := updatePathEnv()
-		if err != nil {
-			return 0
+func FuzzIntegCreateContainerWithTearDown(f *testing.F) {
```

Comment:
This 2 fuzz tests seem already broken (at least on CI), because in github action `$OUT` points to `/github/workspace/build-out/`, not `/out`.

E.g., log from Build Fuzzers: `+ mkdir /github/workspace/build-out/containerd-binaries`
https://github.com/containerd/containerd/actions/runs/12481295870/job/34833535642